src/experimentClient.ts

```diff
@@ -373,6 +382,29 @@ export class ExperimentClient implements Client {
     return this;
   }
 
+  private getInitialFlags(): Record<string, EvaluationFlag> {
+    if (this.config.initialFlags) {
+      const initialFlags = JSON.parse(this.config.initialFlags);
```

Comment:
For the sake of efficiency. We should only JSON parse this string once, and store the result in a class variable to be used in `mergeInitialFlagsWithStorage()`.